gallery/tutorials/configuration.py

```diff
@@ -76,13 +76,33 @@
 #       aspire.config.set_file('/path/to/some_experimental_config.yaml')
 #
 #
-# Or simply set specific variables as needed:
-#
-#     .. code-block:: python
-#
-#       config['nufft']['backend'] = 'cufinufft'
+# Or simply set specific variables as needed.
+# Here we will disable progress bars displayed by
+# ``aspire.utils.trange`` and ``aspire.utils.tqdm``.
 #
 
+import time
+
+from aspire import config
+from aspire.utils import trange
+
+# Progress bars are displayed by default.
+
+print("Disabling progress bars")
+config["logging"]["tqdm_disable"] = True
+
+for _ in trange(3):
+    time.sleep(1)
+print("Done Loop 1\n")
+
+print("Re-enabling progress bars")
+config["logging"]["tqdm_disable"] = False
+
+for _ in trange(3):
+    time.sleep(1)
+print("Done Loop 2\n")
+
```

Comment:
What if `tqdm_enable=True` was the default, to avoid this double negative?

Reply:
I had considered that, but thought it made more sense to match tqdm's `disable` (since we're just passing it through).

I don't really care either way tbh. If you feel strongly about it i can change it real quick. LMK.

Reply:
Ok yeah, I can see that. I also don't feel strongly so let's leave it